0/a-burke/molecule.py

```diff
@@ -0,0 +1,85 @@
+#Author: Ally Burke
+
+
+import sys
+sys.path.append('../../extra-files')
+import copy
+import numpy as np
+from masses import get_charge
+from masses import get_mass
+
+class Molecule(object):
+
+	#Used to initialize an instance of the Molecule class
+
+	def __init__(self, geom_str, units = "Angstroms"):
+		self.units = units				#Specifies units for spatial coordinates
+		self.read(geom_str)				#Reads in the geometry file for the specific molecule	
+		self.natom = self.read(geom_str)		#Specifies the number of atoms in the molecule
```

Comment:
Lines 18 and 19 are a redundant. Line 17 calls the read() function which should read the string and set everything up. Furthermore, read() doesn't return anything so setting a variable equal to self.read() will keep that variable null.